crates/ra_proc_macro/src/lib.rs

```diff
@@ -84,7 +81,7 @@ impl ProcMacroClient {
             ProcMacroClientKind::Process { process, .. } => {
                 let macros = match process.find_proc_macros(dylib_path) {
                     Err(err) => {
-                        eprintln!("Fail to find proc macro. Error: {:#?}", err);
+                        eprintln!("Failed to find proc macros. Error: {:#?}", err);
```

Comment:
```suggestion
                        eprintln!("Failed to find proc macros. Error: {}", err);
```

I usually prefer `Display` for errors.

Reply:
I think that if you don't do this in debug mode you won't get the error chain

Reply:
But yeah, this is not `anyhow::Error`. Actually I still think debug printing this will give more details about the error. I wouldn't argue if such error was an expected/managed one...